main.go

```diff
@@ -260,6 +264,17 @@ func generate(in configInput) ([]byte, error) {
 		c.Node.Metadata["TRAFFIC_DIRECTOR_CLIENT_ENVIRONMENT"] = in.deploymentInfo
 	}
 
+	// Check for include XDS Federation flag to include Authorities in bootstrap config
```

Comment:
Remove this comment? It is stating the obvious.

Reply:
sure. removing it